src/plugins/libgen.py

```diff
@@ -97,6 +97,8 @@ def feed(self, item):
         """
         Feed an item to the bookwyrm instance, otherwise print it out.
         """
+        item = {**item, 'origin_plugin': 'libgen.py'}
```

Comment:
We'll want to change this before merge. Instead of explicitly setting this in every plugin, the interface between Python and C++ should be able to figure this out. See https://github.com/pybind/pybind11/issues/1646.

Reply:
Fixed in squashed 1855331 below.